### ISSUE TITLE
multiple sequence training (inertia question)

### ISSUE THREAD

**Issue by huiyuHarvey** [jmschrei/pomegranate#137]
Hi Jacob,
I have questions about `inertia` parameter in the `fit()` function in Distribution or in HMM,
It means `new_param = old_param * inertia + new_param * (1-inertia)`.
If I fit the model with multiple sequences like this,
```
for f in file_list:
    data = get_data(f)
    model.fit(data, inertia=0.5)
```
does it mean that the sequence for training at the beginning will be less important as the new data?
How can I make all sequence equal important?

And since the `fit()` method in `HMM.pyx` could fit multi-sequences, does `edge_inertia`or `distribution_inertia` works the same as the fit() in Distributions?

**Comment by jmschrei** [comment]
Good question. The way it works is that summary statistics are collected for each sequence and stored in the model in the `model.summarize()` step. Then, parameters are updated (possibly with inertia) using the `model.from_summaries()` step. This means the ordering of the sequences does not matter, even if you ended up calling `model.summarize()` many times because your data could not all fit in memory. `model.fit()` is just a wrapper around summarize and from_summaries calls, since it's an iterative algorithm.

**Comment by huiyuHarvey** [comment]
So it's not the weight of the sequences but the importance of initiative.

**Comment by jmschrei** [comment]
I don't understand what that means.

**Comment by huiyuHarvey** [comment]
`Then, parameters are updated (possibly with inertia) using the model.from_summaries() step.`
I mean if you set `inertia = 1`, it means the parameter would stay the same as initialization; while if `inertia = 0`, during update, the `model.from_summaries()` step would ignore the initialization. Does it kind of reflect the importance of initiative?  Although, as it updates, the importance of initiative would be less.

**Comment by jmschrei** [comment]
It doesn't ignore the initialization. The initialization is directly used to infer new parameters in the expectation step. You can think of it similar to step size in gradient descent.

**Comment by huiyuHarvey** [comment]
Nice explanation!!! Thx!
